### DISCUSSION TITLE
Live Chatbox

### DISCUSSION THREAD

**Discussion by yusbaklee** [h5bp/html5-boilerplate#2691] [General]
How can you use only html code to create a live chat box????

**Comment by yusbaklee** [comment]
Can someone help me with html code for live chat box

**Comment by coliff** [comment]
hey there! I'm not sure exactly what you're trying to do, but maybe this tutorial may help:
https://www.freecodecamp.org/news/building-a-chat-application-with-mean-stack-637254d1136d/

or if not please try searching online for a different way:
https://duckduckgo.com/?q=build+a+live+chat+box&ia=web

**Reply by yusbaklee** [reply]
Oooh, thank you. Will get back to you soon
